glDrawRangeElements references things that don't exist.

NicolBolas opened the pull request:
The `glDrawRangeElements` functions say:

-> mode, start, end, and count match the corresponding arguments to glDrawElements

Of course, `glDrawElements` does not have `start` or `end`. This pull request removes them.

oddhack commented:
Could you re-do this without the spurious indentation / line break deltas? It's a bit hard to review right now. May apply to your other PRs, not sure. Thanks!

NicolBolas commented:
I think I've fixed the formatting issue on this one. I'll check the others for similar issues.

My XML editor apparently decided that it wanted to adjust the formatting of any lines it edits, so I've switched to using a straight text editor.

oddhack commented:
Yeah, I'm kinda hoping to convert these to asciidoc but it's a background task. Docbook looked a lot better in 2006.
